src/backend/expungeservice/crawler/parsers/case_parser.py

```diff
@@ -37,8 +39,20 @@ def feed(data) -> CaseParserData:
             probation_revoked = date.fromdatetime(datetime.strptime(probation_revoked_date_string, "%m/%d/%Y"))
         else:
             probation_revoked = None  # type: ignore
-        return CaseParserData(hashed_charge_data, hashed_dispo_data, balance_due, probation_revoked)
+        return CaseParserData(district_attorney_number, hashed_charge_data, hashed_dispo_data, balance_due, probation_revoked)
 
+    @staticmethod
+    def __parse_district_attorney_number(soup) -> str:
+        district_attorney_key = "District Attorney Number:"
+        labels = soup.find_all("th", "ssTableHeaderLabel", limit=10)
+        table = {}
+        for tag in labels:
+            table[tag.string] = tag.parent.find("td").string
+        if district_attorney_key in table:
+            return table[district_attorney_key]
+        else:
+            return ""
```

Comment:
```suggestion
        DISTRICT_ATTORNEY_KEY = "District Attorney Number:"
        labels = soup.find_all("th", "ssTableHeaderLabel", limit=10)
        table = {tag.string: tag.parent.find("td").string for tag in labels}
        return table.get(DISTRICT_ATTORNEY_KEY, "")
```
Note I have not tested this.

Reply:
I ran the proposed code changes in a local directory and it works fine.

I'm not familiar with how to proceed with changes. Do I click commit suggestion or do I test everything in docker again and do another pull request?

Reply:
Clicking accept commit suggestion works. You can also just commit locally and push the commit (not a new PR).